docs/source/introduction.rst

```diff
@@ -0,0 +1,27 @@
+Introduction
+============
+
+pyOpenMS is a python library for Liquid Chromatography-Mass Spectrometry (LC-MS) data analsis.
+
+This introduction is aimed at users new to the field of LC-MS data analysis and will cover some basics in terms of data generation.
+
+How to handle the data analysis, available data structures, algorithms and more is covered in the various subsections of this documentation.
+
+Liquid Chromatography
+---------------------
```

Comment:
can you run this through a spell checker?

Reply:
Good call, forgot about it!